themes/cardinal_service/pdb_components/opportunities_list/src/Components/SelectList.jsx

```diff
@@ -38,73 +40,89 @@ const Container = styled.div`
   .MuiInputBase-input,
   .MuiChip-root {
     font-size: 16px;
+    font-weight: 400;
     line-height: normal;
-    margin: 5px 0;
+    max-width: 80%;
```

Comment:
with this 80% on the input, it provides an area around the arrow that you can click and the options won't be provided. I was able to fix it in inspector with this on the `<input>` element
```
    max-width: 100%;
    margin: 0;
    padding: 15px;
```

Reply:
Awesome catch! I totally overlooked that 🤦  

I separated the styling for `.MuiAutocomplete-input` and `.MuiChip-root` in addition to implemented the changes for the click-options.